web/assets/js/views/PhysicalEvaluation-0.3.js

```diff
@@ -615,6 +615,15 @@ PMI.views["PhysicalEvaluation-0.3"] = Backbone.View.extend({
                 inches += parseFloat($("#alt-units-height-in").val());
             }
             val = this.inToCm(inches);
+        } else if (type == "alt-units-height-ftin") {
```

Comment:
It seems like you are trying to display the user-entered values in the top right corner. Should we apply this same approach to other fields such as weight, waist circumference, and hip circumference? Also, this entire process needs to be handled for pediatric forms as well.

Reply:
Actually that part of the code is for updating the original entry (height cm in this case) using the converted value of the alternative units. I had to update this particular part just for the height code. The other stuff still works fine. You are right that I need to update the top right display though. I'll fix that.